modules/ROOT/pages/understanding-orchestration-using-mule.adoc

```diff
@@ -8,24 +8,37 @@ Orchestration is how Mule facilitates the process of moving data through a Mule
 
 == Orchestration and Flows
 
-Orchestration involves decoupling point-to-point integration in favor of chains of operations that can be reused or changed as systems or business needs change. It involves moving beyond patterns into a service-based framework, where you wrap multiple components with sources to decouple them from protocols and routing requirements, enabling multiple services to work as one when implementation requirements demand it. Some common use cases that flows easily orchestrate include:
+Orchestration involves decoupling point-to-point integration in favor of chains of operations that can be reused or changed as systems or business needs change. It involves moving beyond patterns into a service-based framework, where you wrap multiple components with sources to decouple them from protocols and routing requirements. This enables multiple services to work as one when implementation requirements demand it. Some common use cases that flows easily orchestrate include:
 
-* simple integration
-* scheduled data processing
-* connecting cloud and on-premise applications
-* event processing where multiple services must be composed and operate simultaneously
+* Simple integration
+* Scheduling data processing
+* Connecting cloud and on-premise applications
+* Event processing, where multiple services must be composed simultaneously and must operate simultaneously
```

Comment:
`composed simultaneously` does not make sense from a technical stand point

Reply:
Fixed in new commit.